src/plugin/leaflet.canvas-markers.js

```diff
@@ -141,6 +145,11 @@ function layerFactory(L) {
 
             map.off('moveend', this._reset, this);
             map.off('resize',this._reset,this);
+
+
+            if (map._zoomAnimated) {
+                map.of('zoomanim', this._animateZoom, this);
```

Comment:
This is a typo which causes errors I believe.

Should me ```map.off```

Reply:
Good pickup, thanks! Will fix that